styles.css

```diff
@@ -723,6 +723,50 @@ label {
   display: none;
 }
 
+/* modal styling */
+#custom-modal {
+  width: 100%;
+  height: 755px;
+  background-color: #3c3a39;
+  position: fixed;
+  top: 0;
+  padding: 20px;
+  z-index: 1;
+}
+
+#modal-delete {
+  display: flex;
+  justify-content: flex-end;
+  padding-top: 10px;
+  font-size: 20px;
+  color: #cdcac9;
+  font-family: 'Inter', sans-serif;
+  cursor: pointer;
+  padding-right: 160px;
+}
+
+#menu-list {
+  list-style-type: none;
+  margin-top: 20px;
+}
```

Comment:
Impressive work so far. We just have to make a few changes to your design so it looks exactly like the Figma template. Kindly consider setting a full `width` for your `ul` tags. This way your `nav` text will be aligned to the `start` of the full `width` of the screen and so will the horizontal line below the text.  Take a look at the screenshot below for more details:

Figma:
<img width="296" alt="Screenshot 2023-04-10 at 15 52 34" src="https://user-images.githubusercontent.com/107192076/230943421-828f1104-9e16-4bdf-9c05-8372f9c930b8.png">

Your design:
<img width="324" alt="Screenshot 2023-04-10 at 15 53 30" src="https://user-images.githubusercontent.com/107192076/230943569-78fa4ad2-6ebc-45b8-b998-b6ba601bdd3b.png">

Reply:
resolved thanks